zlib/tls.go

```diff
@@ -0,0 +1,40 @@
+package zlib
+
```

Comment:
There needs to be

(1) a specific flag validation call
(2) a per-module init call
(3) a per-module per-routine init call